docs/concepts/policy/pod-security-policy.md

```diff
@@ -1,265 +1,521 @@
-1. portworxVolume
-1. scaleIO
-1. storageos
-1. \* (allow all volumes)
-
-The recommended minimum set of allowed volumes for new PSPs are
-configMap, downwardAPI, emptyDir, persistentVolumeClaim, secret, and projected.
-
-### Host Network
- - *HostPorts*, default `empty`. List of `HostPortRange`, defined by `min`(inclusive) and `max`(inclusive), which define the allowed host ports.
-
-### AllowPrivilegeEscalation
-
-Gates whether or not a user is allowed to set the security context of a container
-to `allowPrivilegeEscalation=true`. This field defaults to `false`.
-
-### DefaultAllowPrivilegeEscalation
-
-Sets the default for the security context `AllowPrivilegeEscalation` of a container.
-This bool directly controls whether the `no_new_privs` flag gets set on the
-container process. It defaults to `nil`. The default behavior of `nil`
-allows privilege escalation so as to not break setuid binaries. Setting it to `false`
-ensures that no child process of a container can gain more privileges than
-its parent.
-
-### AllowedHostPaths
-
-This specifies a whitelist of host paths that are allowed to be used by Pods.
-An empty list means there is no restriction on host paths used.
-Each item in the list must specify a string value named `pathPrefix` that
-defines a host path to match. The value cannot be "`*`" though.
-An example is shown below:
+Then the `(Cluster)Role` is bound to the authorized user(s):
 
 ```yaml
-apiVersion: extensions/v1beta1
-kind: PodSecurityPolicy
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: custom-paths
+  name: <binding name>
+roleRef:
+  kind: ClusterRole
+  name: <role name>
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+# Authorize specific service accounts:
+- kind: ServiceAccount
+  name: <authorized service account name>
+  namespace: <authorized pod namespace>
+# Authorize specific users (not recommended):
+- kind: User
+  apiGroup: rbac.authorization.k8s.io
+  name: <authorized user name>
+```
+
+If a `RoleBinding` (not a `ClusterRoleBinding`) is used, it will only grant
+usage for pods being run in the same namespace as the binding. This can be
+paired with system groups to grant access to all pods run in the namespace:
+```yaml
+# Authorize all service accounts in a namespace:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: system:serviceaccounts
+# Or equivalently, all authenticated users in a namespace:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: system:authenticated
+```
+
+For more examples of RBAC bindings, see [Role Binding
+Examples](docs/admin/authorization/rbac/#role-binding-examples). For a complete
+example of authorizing a PodSecurityPolicy, see
+[below](#example).
+
+## Policy Order
+
+In addition to restricting pod creation and update, pod security policies can
+also be used to provide default values for many of the fields that it
+controls. However, this presents a problem when multiple policies are available
```

Comment:
not sure it's a problem if the behavior is defined. rephrase to simply define behavior when multiple policies match?